patches/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java.patch

```diff
@@ -1,14 +1,27 @@
 --- ../src-base/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java
 +++ ../src-work/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java
-@@ -153,6 +153,7 @@
+@@ -72,9 +72,10 @@
+     {
+         ItemStack itemstack = this.field_145945_j.get(4);
+ 
+-        if (this.field_184278_m <= 0 && itemstack.func_77973_b() == Items.field_151065_br)
++        int fuel = net.minecraftforge.event.ForgeEventFactory.getBrewingStandFuel(itemstack);
++        if (this.field_184278_m <= 0 && fuel > 0)
```

Comment:
Tested and looks good to me, but you may want to change this logic so that you can add fuel when 
`fuel > 0 && this.field_184278_m + fuel <= 20` so that small fuels can be used when the brewing stand fuel value is low.
Right now if the brewing stand's fuel value is 1, you can't add even 1 more fuel.

Reply:
Should I add another specual check to allow fuels with 20+ values?

Reply:
I don't think we should allow fuel with higher than 20 value, that's as much as the brewing stand can handle.

Reply:
I don't think there is really a limit. Vanilla can work with fuel more than 20. It just renders a full bar. There may be vanilla-based maps where there are brewing stands with a very large number of fuel, so it is virtually impossible to consume all the fuel. (that's the usecase I may think of) Mod added fuels and mod brewing stands may have different standards. I've created a strawpoll.me entry here: http://www.strawpoll.me/14691727 I will post it on your discord for discussion (or mmd)

Reply:
We should just follow vanilla: If vanilla clamps it to 20 on load from disk then we should set that as the limit, if it doesn't, then I don't see why there should be one

Reply:
Is there any way to control how the brewing stand "consumes" the fuel? If that can be changed I'd say just leave this at 20 and let people make a damageable item that takes damage on consumption if they want bigger.

Reply:
1. Vanilla does not clamp. It is just a byte value.
2. The brewing stand always shrinks the stack size by one.

Reply:
I assumed it was clamped at 20. It's silly to go over 20 but if it works then why not?
Also, this kind of thing shouldn't really go to a vote... it's better to just discuss it.

Reply:
I think knightminer is asking what the units on "20" are. iirc it's how many brewing operations the stand can complete before it needs to be refueled

Reply:
No, I was essentially asking if I use a flint and steel as a fuel, can I have the flint and steel take one damage and provide 4 fuel while remaining in the slot.

Reply:
ah, no, vanilla always consumes 1 whole item from the stack and adds a specific number of "brews" to the stand